tests/publiccloud/slem_basic.pm

```diff
@@ -114,6 +116,26 @@ sub run {
     $instance->run_ssh_command(cmd => 'sudo sestatus | grep enabled');
     $instance->run_ssh_command(cmd => 'sudo dmesg');
     $instance->run_ssh_command(cmd => 'sudo journalctl -p err');
+
+    # volume size tests
+    if (get_var('PUBLIC_CLOUD_ROOT_DISK_SIZE')) {
```

Comment:
this part is about checking the result , but I would expect something which would instruct CSP to give bigger block device so this would trigger root FS resize . Some tweak of terafform profile and/or creat_instance logic to pass some param :thinking:  or am I missing something obvious ?

Reply:
`PUBLIC_CLOUD_ROOT_DISK_SIZE` instructs terraform to create disk on desired size in GB, logic to instance code was added about month ago ?